lua/swampshop/sh_products.lua

```diff
@@ -83,28 +83,52 @@ function SS_WeaponProduct(product)
         self:SS_WeaponProduct_OnBuy(ply)
     end
 
+    function product:CannotBuy(ply)
```

Comment:
Can you make it select the weapon? So a bind can either buy the weapon, or just select it

Reply:
Will it still charge the player?

Reply:
Select weapon then return a string, and it won't buy it

Reply:
oh okay